### PR TITLE
Addition of a requirements.txt file

### PR DESCRIPTION
**Pull request by tomaszwasowicz-digica**
Added a requirements.txt file

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1,212 @@
+absl-py==0.14.1
+alabaster==0.7.8
+alembic==1.4.1
+antlr4-python3-runtime==4.8
+apturl==0.5.2
+attrs==21.2.0
+Babel==2.6.0
+backports.entry-points-selectable==1.1.0
+beautifulsoup4==4.10.0
+black @ git+git://github.com/psf/black@2f3fa1f6d0cbc2a3f31c7440c422da173b068e7b
```
Comment (pawelpreczynski-digica) [inline requirements.txt:10]: Why to we refer to this [commit](https://github.com/psf/black/commit/2f3fa1f6d0cbc2a3f31c7440c422da173b068e7b)?
Shouldn't we refer to some version? like `21.10b0` or `stable`...

**Review by pawelpreczynski-digica — CHANGES_REQUESTED** [review]

File: requirements.txt (hunk 1 above)
Comment (tomaszwasowicz-digica) [reply requirements.txt]: Please check for an updated push. I've recreated the requirements.txt file strictly from the docker files instructions.

**Review by pawelpreczynski-digica — APPROVED** [review]
Looks good!

**Comment by tomaszwasowicz-digica** [comment]
creating a virtual environment from requirements file was successful. To run the file you need to use the command: 

virtualenv venv && venv/bin/pip install -r requirements.txt -f https://download.pytorch.org/whl/cu113/torch_stable.html

which creates the virtual environment, install the module packs from the requirements.txt file  ( also searching for the torch==1.10.0+cu113 torchvision==0.11.1+cu113 torchaudio==0.10.0+cu113 pack modules in the added address line )

The run instruction for the requirements.txt file will be added to the new readme file.

**Pull request merged**